docs/content/docs/deep/dataobject-aqueduct.md

```diff
@@ -15,7 +15,7 @@ how to combine those distributed data structures with custom code (business logi
      argument) and then run "npm run build:md-magic" from the docs folder. That will update all autogenerated
      content, so you should commit the resulting changes. -->
 
-<!-- AUTO-GENERATED-CONTENT:START (INCLUDE:path=../../../../packages/framework/aqueduct/README.md&start=19&end=-27) -->
```

Comment:
For reviewers: these line numbers were outdated with the new template contents in the `aqueduct` README. Also discovered a trailing line that was previously (incorrectly) omitted.